ADR/0025-container-images.md

```diff
@@ -0,0 +1,94 @@
+# 25. Container Image Process
+
+Date: 2023-05-26
+
+## Status
+
+Proposed
+
+## Context
+
+The purpose of this document is to establish container image management practices for AppStudio container images that are deployed in the staging and production environments.  The goal is to ensure that AppStudio is continuously maintaining secure operations that are in accordance with the ESS SEC-PATCH-REQ-2 (OS Patching) requirements.
+
+### Scope
+* The scope of this process is limited to the images found in our [quay.io/organization/redhat-appstudio](https://quay.io/organization/redhat-appstudio) repository.
```

Comment:
Images produced via the build platform will not be pushed to this organization by default. How will images get pushed there? Will there be a required release pipeline process for them to follow to get the images mirrored?

Reply:
Good point.  Instructions have been pretty vague or word of mouth.  The only instructions I know of are to follow https://redhat-appstudio.github.io/infra-deployments/docs/deployment/extending-the-service.html, copy over the tekton yamls from some other component which and customize them.  I don't know if there are better instructions anywhere else that I can link to.

Reply:
Should we get another issue opened so someone can write up instructions that we can link to from this ADR?

Reply:
There is a related conversation happening now about this process. Maybe we can combine/unify the two threads.

Reply:
Any update on this thread?

Reply:
I was on PTO when the conversation happened, so I don't know much about what was discussed.   We can consider accepting what's currently written and update this ADR when there's more info

Reply:
I just requested a review on this PR from @sbose78 and @scoheb on this document as they have been involved in the effort to build using the platform.

Reply:
@sbose78 and @scoheb I'm going to merge this but can you submit a PR to update it if you have later/better information to add?